### ISSUE TITLE
when awin_or_fsize=800，getting bad result

### ISSUE THREAD

**Issue by JK1532** [Jonathan-LeRoux/lws#14]
HI, Jonathan-LeRoux. Thank you for your work，this repo helps me a lot，but when awin_or_fsize=800, hop_length=160, fftsize=1024, the output of istft is getting worse.
the origin：
![origin](https://user-images.githubusercontent.com/23287555/106569359-5ebaae80-656f-11eb-99a5-02623ef5725f.jpg)
the reconstruction:
![reconstruction](https://user-images.githubusercontent.com/23287555/106569511-97f31e80-656f-11eb-9255-0e8822f1fa6f.jpg)

**Comment by Jonathan-LeRoux** [comment]
Thanks for reporting this bug, I was able to reproduce it on my end.
It looks like LWS degrades the signal significantly if the hop size does not divide the FFT size.
When I added support for hop sizes not dividing window lengths (see [this issue](https://github.com/Jonathan-LeRoux/lws/issues/10)), I guess I only tested cases where the FFT was padded to a multiple of the hop size... which is not super practical, I agree. Let me see if that's fixable. I thought I had found the issue, related to rounding of the ratio of frame size to frame shift, but fixing that didn't fix the poor reconstruction. I'll let you know if I make progress.
